compose-desktop/src/main/kotlin/main.kt

```diff
@@ -36,6 +54,11 @@ private val dateService = JvmDateService()
 fun main() = application {
     val windowState = rememberWindowState()
 
+    LaunchedEffect(key1 = this) {
```

Comment:
@joreilly I have no clue if that is the "correct way" to do it on Compose Desktop. Suggestions?

Reply:
I'm definitely no expert on Compose for Desktop but that seems ok to me at least